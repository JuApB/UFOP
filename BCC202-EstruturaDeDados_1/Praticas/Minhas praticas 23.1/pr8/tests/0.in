2
5
pedro 29 3
daiane 30 1
guilherme 25 1
tales 12 1
joao 81 3
2
carla 35 2
paula 22 3
