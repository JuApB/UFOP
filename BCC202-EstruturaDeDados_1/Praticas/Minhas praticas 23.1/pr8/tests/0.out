joao 81 3
pedro 29 3
paula 22 3
tales 12 1
daiane 30 1
guilherme 25 1
carla 35 2
